README.md

```diff
@@ -23,17 +25,40 @@ docker-compose up --build
 You can check that the connection with twitter is working by executing:
 
 ```
-curl "http://localhost:5000/twitter/get_tweet_data/974276383051583488" -v
+curl "http://localhost:5000/twitter/get_tweet_data/947121935690944512" -v
 ```
 
 And you can check that the connection with neo4j is working by executing:
 
 ```
-curl "http://localhost:5000/neo4j/get_user_info/25073877" -v
+curl "http://localhost:5000/neo4j/get_user_info/870716233393614849" -v
 ```
 
 You can find all other API endpoints on the `server.py` file.
 
+## Crawling
+
+Every time you call the /twitter/get_tweet_data endpoint, it populates the database with information about the tweet, but not the user followers connections. In order to populate this for a user you will need to call this endpoint:
```

Comment:
it does not populate the database, its just requesting twitters api from the backend and hands this information to the front-end.
Populating the database with tracemap information is not yet implemented